opensrp-giz-malawi/src/main/res/values/strings.xml

```diff
@@ -102,4 +102,5 @@
     <string name="action_start_sync">Sync Started</string>
     <string name="all_registers">All Registers</string>
     <string name="menu_child_clients">Child Clients</string>
+    <string name="due_only">Due Only</string>
```

Comment:
Fix the position of the due Only Switch & Add the register name at the top of the register

Reply:
Look at the ANC design for what I am talking about. Or countercheck with the actual design

Reply:
updated on https://github.com/OpenSRP/opensrp-client-giz-malawi/pull/70

Reply:
ok